test/utils/FeatureChecker.test.ts

```diff
@@ -1,4 +1,4 @@
-import { expect } from 'chai';
+import { assert } from 'chai';
```

Comment:
We tend to user chai/expect. I do not quite see point for making that change.

Reply:
I made this change originally when I added that `getEnabledReasons` function, because I noticed all of the TSLint issues with the existing tests. I figured since both test methods are from the same library, the change was a more elegant solution than simply disabling the linter.

Reply:
I will use consistency card this time. If you do not like suppressing linter warning you could use `be.equal(true)`.

Reply:
Sure, I'll revert the change then.